src/features/classification/classificationColorsMap.js

```diff
@@ -0,0 +1,58 @@
+// @flow
+import messages from './messages';
+import {
+    CLASSIFICATION_COLORID_0,
```

Comment:
I think these should be `CLASSIFICATION_COLOR_ID_0` since id is a separate word

Reply:
+1 on that